akka-sample-distributed-data-java/README.md

```diff
@@ -74,23 +74,6 @@ The multi-node test for the `ShoppingCart` can be found in [ShoppingCartSpec.sca
 
 Read the [Consistency](http://doc.akka.io/docs/akka/2.6/java/distributed-data.html#Consistency) section in the documentation to understand the consistency considerations.
```

Comment:
Looks like all doc links should be updated, should be like: https://doc.akka.io/docs/akka/2.6/typed/distributed-data.html#consistency